docs/spec/inflation/end_block.md

```diff
@@ -0,0 +1,57 @@
+# End Block
+
+```
+EndBlock() 
+    processProvisions()
+```
+
+# Inflation
+
+Validator provisions are minted on an hourly basis (the first block of a new
+hour). The annual target of between 7% and 20%. The long-term target ratio of
+bonded tokens to unbonded tokens is 67%.
```

Comment:
Would it be possible to describe why the exact target of 67% in a concise manner?

Reply:
Agreed - or cite the whitepaper / relevant source.

Reply:
I disagree this IS the spec, it supersedes the white paper - it's not a place where choices always need to be justified. This number for instance comes out of a bunch of conversations with Jae - there is no good reason it just seems like a fine value. For y'alls record if the value is to small its a security concern, if it's to large then there isn't enough liquidity for price discovery -> 67% seems fine.